docs/faq.md

```diff
@@ -87,7 +87,7 @@ take the aggregation type [here](https://docs.microsoft.com/en-us/azure/azure-mo
 
 Modes:
 * `Default` mode **requires** hub server. When there is no server connection available for the hub, the client tries to connect to this hub fails.
```

Comment:
How about mention that `Default` mode can with with RestApi to break the "user impression" of binding `Serverless` with RestApi?